### DISCUSSION TITLE
[How to move extensions to my 2nd monitor?]

### DISCUSSION THREAD

**Discussion by Cubuss** [home-sweet-gnome/dash-to-panel#1903] [Q&A]
Hey im using dash to panel and have some extensions installed wich i wish to be displayed on my 2nd montiro instead of my main montior the extensions are embeded in Right and Center box
I know this cause when i click visible it makes the extensions disapear,

![image](https://github.com/home-sweet-gnome/dash-to-panel/assets/9220880/abe3c495-edb3-4ad9-b726-4a0e5a4e05f3)

Is this even possible?

**Comment by Cubuss** [comment]
will it be possible to make this extension compatible with this? https://github.com/Noobsai/fullscreen-avoider this does exactly the above it will move the main panel with all the extension if there is an application that goes full screen

**Comment by charlesg99** [comment]
Hello, the default gnome right and center boxes are displayed on the main panel, so you can set it to the monitor where you'd like to have your extension icons.

![image](https://github.com/home-sweet-gnome/dash-to-panel/assets/33044886/4bae27b1-989b-4cca-b06e-a41e17a6c725)

AFAIK the fullscreen-avoider should somewhat work with dash-to-panel, but it would need a small modification to account for and move the panelBox's parent if it has one (dash to panel adds it while default gnome doesn't have one). Also not sure what would happen if you have multiple panels and this extension moves the primary panel to a monitor already displaying a secondary panel.

**Comment by Cubuss** [comment]
hey thank you for your response if i change display the main panel to 2nd monitor it will move everything tough including ark menu and other extensions like GSConnect WIreless HID all show on 2nd monitor now aswel , i did saw option in ArcMenu to display it on all monitors, would it be possible to add feature to move for example only the center box to different monitor
